generators/generator-multibot/package.json

```diff
@@ -1,12 +1,11 @@
 {
-  "name": "generator-multibot",
-  "version": "1.0.6",
-  "description": "Preview conversational core package for TESTING ONLY",
-  "main": "./generators/app/templates/ConversationalCore/index.js",
+  "name": "@microsoft/generator-microsoft-bot-multibot",
+  "version": "0.0.1",
+  "description": "Preview multi-bot template for TESTING ONLY",
+  "main": "./generators/app/index.js",
   "dependencies": {
-    "generator-microsoft-bot-adaptive": "^1.0.0-preview-20210219.c75d357",
-    "generator-microsoft-bot-conversational-core": "^1.0.0-preview-20210219.916e9f9",
-    "generator-preview-calendar-skill": "file:../generator-preview-calendar-skill",
+    "@microsoft/generator-microsoft-bot-adaptive": "^1.0.0-preview-20210219.c75d357",
+    "@microsoft/generator-microsoft-bot-calendar": "file:../generator-preview-calendar-skill",
```

Comment:
At some point someone in chat mentioned moving this one to referencing the published calendar skill. But not sure what's the current plan of record on that front.

Reply:
fixed!